_docs_integrate/01-integrate-scenarios.md

```diff
@@ -15,35 +15,39 @@ Here you'll find everything you need to seamlessly dive into the world of enmesh
 
```

Comment:
There seems to be a word missing in the previous sentence.

Reply:
Which sentence do you mean exactly and what would be your suggestion for improvement?

Reply:
"Whether you're just starting out or **are** already familiar with enmeshed"